app/prover/[teamId]/page.tsx

```diff
@@ -221,63 +199,122 @@ export default async function ProverPage({ params }: ProverPageProps) {
           {team.github_org && (
             <HeroItem className="hover:underline">
               <Link
-                target="_blank"
-                rel="noopener noreferrer"
+                hideArrow
+                className="text-body"
                 href={new URL(team.github_org, "https://github.com").toString()}
               >
-                <HeroItemLabel className="text-body">
-                  <GitHub className="text-body-secondary" />
+                <HeroItemLabel className="gap-x-2 text-body">
+                  <GitHub />
                   {team.github_org}
                 </HeroItemLabel>
               </Link>
             </HeroItem>
           )}
         </HeroBody>
-      </HeroSection>
+      </div>
 
-      <section>
-        <h2 className="flex items-center gap-2 text-lg font-normal text-primary">
-          <TrendingUp /> Prover performance
-        </h2>
-        <div className="flex flex-wrap gap-x-8">
-          {performanceMetrics.map(({ key, label, description, value }) => (
-            <MetricBox key={key}>
-              <MetricLabel>
-                <MetricInfo label={label}>{description}</MetricInfo>
-              </MetricLabel>
-              <MetricValue>{value}</MetricValue>
-            </MetricBox>
-          ))}
+      <div className="mt-14 space-y-20">
+        <div className="grid grid-cols-1 gap-6 md:grid-cols-2">
+          <div className="relative rounded-[1.25rem] bg-background p-6">
+            <div className="absolute -inset-px z-[-2] rounded-[calc(1.25rem_+_1px)] bg-gradient-to-tl from-primary to-primary/10" />
```

Comment:
In a diff pr we should use the Card for this to share the same styles (like in the dashboard)